packages/docs/site/i18n/ja/docusaurus-plugin-content-docs/current/blueprints/tutorial/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: ブループリント 101
+description: ブループリントの短期コースへの入り口
```

Comment:
Recently, I started to add more meaningful descriptions to help with SEO You can do the same with 120 characters

Reply:
Thank you.
The current description is "Entrance to a short course on Blueprints."
What kind of description do you have in mind?
https://github.com/WordPress/wordpress-playground/blob/2800a95d10f87302a6d523caec6c75fc0886015c/packages/docs/site/docs/blueprints/tutorial/index.md?plain=1#L3

Reply:
The landing page for the "Blueprints 101" crash course. This short tutorial covers everything you need to get started.